app/src/components/Provinces.tsx

```diff
@@ -0,0 +1,50 @@
+import { Province } from "./Types";
+
+const provinces: Province[] = [
+  { label: "Alberta", id: "ab", region: "Prairies", region_id: "pr" },
+  {
+    label: "British Columbia",
+    id: "bc",
+    region: "British Columbia",
+    region_id: "bc",
+  },
+  { label: "Manitoba", id: "mb", region: "Prairies", region_id: "pr" },
+  { label: "New Brunswick", id: "nb", region: "Atlantic", region_id: "at" },
+  {
+    label: "Newfoundland and Labrador",
+    id: "nl",
+    region: "Atlantic",
+    region_id: "at",
+  },
+  {
+    label: "Northwest Territories",
+    id: "nt",
+    region: "Territories",
+    region_id: "te",
+  },
+  { label: "Nova Scotia", id: "ns", region: "Atlantic", region_id: "at" },
+  { label: "Nunavut", id: "nu", region: "Territories", region_id: "te" },
+  { label: "Ontario", id: "on", region: "Ontario", region_id: "on" },
+  {
+    label: "Prince Edward Island",
+    id: "pe",
+    region: "Atlantic",
+    region_id: "at",
+  },
+  { label: "Quebec", id: "qc", region: "Quebec", region_id: "qc" },
+  { label: "Saskatchewan", id: "sk", region: "Prairies", region_id: "pr" },
+  { label: "Yukon", id: "yt", region: "Territories", region_id: "te" },
+];
+
+export const getProvincesWithNA: () => Province[] = () => {
+  return provinces.concat({
+    label: "N/A",
+    id: "na",
+    region: "N/A",
+    region_id: "na",
+  });
+};
+
+export const getProvinces = () => {
+  return provinces;
+};
```

Comment:
Why are these functions, rather than directly exporting the 2 arrays as constants?

Reply:
It was to avoid having all the lines duplicated just to have one with NA and one without. I added getProvinces to make the calls coherent.
But maybe my reasoning wasn't right I don't know. I'm always trying to avoid duplicated code when I can usually.

Reply:
What do you think about a solution where we remove getProvinces but we keep getProvincesWithNA ?